### PR TITLE
Support directories on CLI

### PR DESCRIPTION
**Pull request by silverwind**
- Fixes: https://github.com/jedmao/eclint/issues/83
- Add: file ignore support on user define globs

### CODE REVIEW

**Comment by gucong3000** [comment]
> The new usage string is not localized.
@silverwind localized now.

File: lib/cli.ts [hunk 1]
```diff
@@ -199,7 +213,7 @@ function infer(yargs: Argv): Stream {
 }
 
 yargs
-	.usage(i18n('Usage: $0 <command> [files...] [options]'))
+	.usage(i18n('Usage: $0 <command> [files,dirs...] [options]'))
```
Comment (gucong3000) [inline lib/cli.ts]: I am not sure. mybe should be `'Usage: $0 <command> [globs...] [options]'`

**Comment by jedmao** [comment]
Thanks @gucong3000. My hands are full with a newborn 👶.

**Comment by gucong3000** [comment]
@jedmao Congratulations!

File: lib/cli.ts (hunk 1 above)
Comment (silverwind) [inline lib/cli.ts]: Hmm it might be that some users are not familiar with the term 'glob' so I thought to keep it simple.

Comment (gucong3000) [inline lib/cli.ts]: But it is a invalid parameter name for `yargs`

Comment (gucong3000) [inline lib/cli.ts]:
```
Commands:
  check [files...]  Validate that file(s) adhere to .editorconfig settings
  fix   [files...]  Fix formatting errors that disobey .editorconfig settings
  infer [files...]  Infer .editorconfig settings from one or more files
```

Comment (silverwind) [inline lib/cli.ts]: Okay with me.

**Pull request merged**